libvast/vast/system/spawn_node.hpp

```diff
@@ -13,14 +13,16 @@
 
 #pragma once
 
-#include <caf/fwd.hpp>
-
 #include "vast/fwd.hpp"
 
+#include "vast/scope_linked.hpp"
+#include "vast/system/actors.hpp"
+
 namespace vast::system {
 
 /// Spawns a new VAST node.
-caf::expected<scope_linked<caf::actor>> spawn_node(caf::scoped_actor& self,
-                                                   const caf::settings& opts);
+// FIXME: make this return a node_actor
```

Comment:
Do you plan to address this in a separate PR?

Reply:
That's actually an oversight, will push another commit shortly.